showpass/Showpass-E2E-tests
# TA-53 Verify a ticket package order via UI(31-0341-465e-a1a7-910944b23fff)

**Description:**
This spec will be verifying a particular ticket package details. This order is linked to the following user:  [user-packages@mailinator.com]

Steps:
- Navigate to https://beta.showpass.com/account/my-orders/31-0341-465e-a1a7-910944b23fff/
- Make sure the following order shows up (Order ID: 31-0341-465e-a1a7-910944b23fff)
- Make sure the front-end order details are correct

**Jira ticket:**
https://showpass.atlassian.net/jira/software/projects/TA/issues/TA-53

**How to test:**
Run the following test suite and ensure the tests pas:
cypress/e2e/user-settings/my-orders.cy.js

**Test execution results:**

<img width="1714" alt="TA-53-tests-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/a0838287-49b3-444b-9cd9-e67ae29949fb">
